frontend/src/components/myPermissions/MyPermissions.vue

```diff
@@ -0,0 +1,12 @@
+<script setup lang="ts">
+import myPermissionsTable from '@/components/myPermissions/myPermissionsTable.vue';
+import PageTitle from '@/components/common/PageTitle.vue';
+</script>
+
+<template>
+    <PageTitle
+        title="My permissions"
+        subtitle="Check all the applications and permissions that you have access to"
+    />
+    <myPermissionsTable />
```

Comment:
Hi Jota, component name needs to use capital letter, MyPermissionTable

Reply:
oh, my bad! I'll fix it, thanks!